cms/templates/widgets/footer.html

```diff
@@ -4,7 +4,15 @@
 <div class="wrapper-footer wrapper">
   <footer class="primary" role="contentinfo">
     <div class="colophon">
-      <p>&copy; ${settings.COPYRIGHT_YEAR} <a href="${marketing_link('ROOT')}" rel="external">${settings.PLATFORM_NAME}</a>. ${ _("All rights reserved.")}</p>
+      <p>&copy; ${settings.COPYRIGHT_YEAR} <a href="${marketing_link('ROOT')}" rel="external">${settings.PLATFORM_NAME}</a>.</p>
+      ## Site operators: Please do not remove this paragraph! This attributes back to edX and makes your acknowledgement of edX's trademarks clear.
+      <p>
+	## Translators: 'EdX', 'edX', 'Studio', and 'Open edX' are trademarks of 'edX Inc'. Please do not translate any of these trademarks and company names.
```

Comment:
This copy came directly from Jennifer, without a comma. I will double-check with her.